examples/Example17/AdeptIntegration.cu

```diff
@@ -234,6 +249,7 @@ void AdeptIntegration::InitializeGPU()
     gpuState.allmgr_d.trackmgr[i]  = gpuState.allmgr_h.trackmgr[i]->ConstructOnDevice();
     gpuState.particles[i].trackmgr = gpuState.allmgr_d.trackmgr[i];
     COPCORE_CUDA_CHECK(cudaMalloc(&gpuState.allmgr_d.leakedTracks[i], QueueSize));
+    InitLeakedQueues<<<1, 1>>>(gpuState.allmgr_d, kCapacity);
```

Comment:
If you want to call this here, the kernel must only construct the current leaked queue, not all three of them.

Reply:
removed from this PR